README.md

```diff
@@ -440,3 +440,2967 @@ but it can be removed and re-generated later, if needed.
 
 ## Contribution
 See our [Contributing Policy](CONTRIBUTING.md) and [Code of Conduct](CODE_OF_CONDUCT.md)
+
+# Kotlin Coding Style Guide (Diktat Code Style), v.1.0.0
```

Comment:
why link to the existing code style from `info` directory is not enough and why do we need to copy paste it here?

Reply:
Because of three reasons, I got a great suggestion from several smart people, who needed this code style but were not able to find it:
1) Google search mechanisms do not index internal code, only readme; 
2) I also created a WIKI page, it is also indexed;
3) The name was incorrect, it should be instead of Diktat Code Style -> Kotlin Code Style